### PR TITLE
Design specific project page

### LINKED ISSUE #27: Design specific product page
Design specific product page according to design sketches.

### PR DESCRIPTION
**Pull request by emilps**
What's done:
- Design according to sketches
- Added correct fields to cms
- Changed the projectroll according to cms changes

closes #27

### CODE REVIEW

File: src/templates/index-page.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const IndexPageTemplate = ({
   <div>
     <IndexImage />
     <section className="section section--gradient">
+      {console.log(image)}
```
Comment (eirikdahlen) [inline src/templates/index-page.js]: Remove this one

File: src/templates/project.js [hunk 2]
```diff
@@ -5,54 +5,105 @@ import Helmet from 'react-helmet';
...
   title: PropTypes.string,
+  seconddescription: PropTypes.string,
+  secondtitle: PropTypes.string,
+  thirddescription: PropTypes.string,
+  thirdtitle: PropTypes.string,
   helmet: PropTypes.object,
-  tags: PropTypes.array
+  mainimage: PropTypes.any,
+  firstminiatureimage: PropTypes.any,
+  secondminiatureimage: PropTypes.any
```
Comment (eirikdahlen) [inline src/templates/project.js]: Vet ikke om any er helt lov å bruke?

File: src/cms/preview-templates/ProjectPreview.js [hunk 3]
```diff
@@ -2,20 +2,29 @@ import React from 'react';
...
-);
+const ProjectPreview = ({ entry }) => {
+  const entryBaseProducts = entry.getIn(['data', 'relatedproducts']);
+  const baseProducts = entryBaseProducts ? entryBaseProducts.toJS() : [];
+  return (
+    <ProjectTemplate
+      description={entry.getIn(['data', 'description'])}
+      title={entry.getIn(['data', 'title'])}
+      description1={entry.getIn(['data', 'description1'])}
+      title1={entry.getIn(['data', 'title1'])}
```
Comment (eirikdahlen) [inline src/cms/preview-templates/ProjectPreview.js]: What is the difference between these?

File: src/templates/project.js [hunk 4]
```diff
@@ -5,54 +5,158 @@ import Helmet from 'react-helmet';
...
+                    ) : (
+                      <Img
+                        fluid={miniatureimage1.childImageSharp.fluid}
+                        alt={`featured image thumbnail for project ${title1}`}
+                        style={{ maxHeight: '270px', maxWidth: '380px' }}
+                      />
+                    )}
+                  </div>
+                ) : null}
+                {miniatureimage2 ? (
```
Comment (eirikdahlen) [inline src/templates/project.js]: Should have onClick link for the whole picture, not just the title of the product?

File: src/templates/project.js [hunk 5]
```diff
@@ -5,54 +5,158 @@ import Helmet from 'react-helmet';
...
+                      />
+                    )}
+                    <hr
+                      style={{
+                        backgroundColor: 'black',
+                        width: '50%',
+                        height: '1px'
+                      }}
+                    ></hr>
+                    <Link
```
Comment (eirikdahlen) [inline src/templates/project.js]: Should be a component? And what happens if it is less than 3 related products?

File: src/templates/project.js (hunk 4 above)
Comment (emilps) [reply src/templates/project.js]: Consider it done

File: src/cms/preview-templates/ProjectPreview.js (hunk 3 above)
Comment (emilps) [reply src/cms/preview-templates/ProjectPreview.js]: It's the first and second description and title.

**Pull request merged**